jwst/extract_2d/extract_2d_step.py

```diff
@@ -17,6 +17,7 @@ class Extract2dStep(Step):
         source_ids = force_list(default=None)     # source ids to be extracted
         source_ra = force_list(default=None)  # source RAs to be extracted, WFSS only
         source_dec = force_list(default=None)  # source DECs to be extracted, WFSS only
+        source_max_sep = float(default=2)  # maximum separation in arcseconds around source_ra, source_dec, WFSS only
```

Comment:
This is a nitpick, but I prefer to make it clear that the default value is a float
```suggestion
        source_max_sep = float(default=2.0)  # maximum separation in arcseconds around source_ra, source_dec, WFSS only
```

Reply:
Fixed by https://github.com/spacetelescope/jwst/pull/10078/commits/51a7e2933ba1a35618e79104d9d3f6ab0078ff9a